src/shlex.ts

```diff
@@ -77,11 +77,7 @@ export function quote(str: string, opt?: ShlexOptions): string {
     if (str === '') {
         return '""';
     }
-    if (!/[\s]/g.test(str)) {
-        // Don't quote if the string doesn't have whitespace
-        return str;
-    }
-    if (/[^\w@%\-+=:,./|]/.test(str)) {
+    if (/[^\w@%\-+=:,./|><]/.test(str)) {
```

Comment:
We can take this change since it shouldn't regress the other fix and will put us back in a state where it should work for the new bug report, but we'll need to update the changelog.  Can you add a line in the changelog for this?

I asked the customer for more information about their configuration so that hopefully we can reproduce the actual problem.